### DISCUSSION TITLE
Not appearing to be near my school

### DISCUSSION THREAD

**Discussion by IlSoloLuke** [community/community#61003] [GitHub Education]
### Hi everyone,

I've tried for over a month to verify my academic status to be able to apply to Github's benefits. I've always received errors when I sent them. This time I correctly set everything up, but I continue to receive the same error which is not true. 
Github does not find my school near my location, even though I live 4km away from it.

Can you help me? (I'm not using any VPN)

**Comment by samus-aran** [comment]
Hi @IlSoloLuke 👋 

Thanks for raising this ⭐ We'll investigate this particular school's details and let you know how this proceeds. We'll post in this thread when we have an update 👍

**Reply by EncryptMan** [reply]
Please also check my application. I am also getting rejected because of GitHub unable to find campus near my location. I live very close to my school.

**Reply by samus-aran** [reply]
Hi @EncryptMan  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by EncryptMan** [reply]
Many days passed and github support team did not replied me back. Before opening 'others' ticket I also opened another ticket almost a month ago and did not got any reply back.

**Comment by IlSoloLuke** [comment]
Thanks, but the problem now is:

because I tried so many times sending the application, it recognizes the proofs i send as "already used ones", what do you suggest?

**Comment by hoper212** [comment]
Thanks
